src/components/ActionsMenuLayout/ActionsMenuLayout.tsx

```diff
@@ -75,6 +100,16 @@ export class ActionsMenuLayout extends React.Component<ActionsMenuLayoutProps> {
     );
   }
 
+  _onListItemClick = (e: MouseEvent) => {
+    const indexValue = Object.values(this.liRefs).findIndex(
+      (i) => i.liRef?.current === e.currentTarget,
+    );
+
+    if (indexValue >= 0) {
+      this._focusItem(indexValue);
+    }
+  };
+
```

Comment:
Theoretically the `ActionsMenu` component is a list of action items, so that each one of the list item is clickable, and sends an action. 
After that, the `ActionsMenu` will usually disappear (it should be inside of a popover).
So I'm not sure we should control the focus.
I think the focus should work the same as hover, meaning they should probably share the same state.

Reply:
@jonathanadler this is fixing the behaviour that was already present. I agree on your point. In our product, we use this component as you described, and item focus state in this scenario doesn't make sense. But I'm not aware of all the use-cases, perhaps the element is used in some scenarios where it's needed. Let me know how to proceed, I'll remove this functionality if it's the right way for this component.

Reply:
🤔  I understand what you're saying.
Although I think it's best to avoid adding this behaviour at the moment.
We can maybe try to add it in a different PR.

In addition, we're also planning on developing an `ActionMenu` component, which will use this `ActionsMenuLayout` component as its content.
It will also already contain a popover (for desktop) and a mobile drawer (for mobile of course).
The `ActionMenu` will handle all the focus and a11y issues, so it will be much easier for consumers to use the component.